tests/installation/partitioning_firstdisk.pm

```diff
@@ -22,7 +22,18 @@ sub take_first_disk_storage_ng {
     return unless is_storage_ng;
     send_key $cmd{guidedsetup};    # select guided setup
     assert_screen 'select-hard-disks';
-    assert_and_click 'hard-disk-dev-sdb-selected';    # Unselect second drive
+    if (check_var('VIDEOMODE', 'text')) {
+        assert_screen 'hard-disk-dev-sdb-selected';
+        if (match_has_tag 'hotkey_d') {
+            send_key 'alt-d';
+        }
+        elsif (match_has_tag 'hotkey_e') {
+            send_key 'alt-e';
+        }
```

Comment:
As you wrote in #3594

> I saw hotkeys differs for the tests run on local beijing openqa server and openqa.suse.de. So branching for the two cases.

Are you sure you tested the same product build from both servers? I doubt that only the PXE server itself would cause this difference.

I recommend to just define the shortcut as is needed based on product versions if there is any necessary difference at all.

Reply:
Yes, the same version. But from different openqa server and different pxe server, beijing pair vs germany pair.

Reply:
Do you think I still need to rerun on latest build as suggested in https://progress.opensuse.org/issues/25510? Beijing only has the latest sle15 alpha4.

Reply:
I don't think that's necessary. We have a test on openQA on the latest build that shows that there is no ncurses problem here it seems: https://openqa.suse.de/tests/1184538#step/partitioning_firstdisk/1

Reply:
textmode@64bit-ipmi has no problem for check_screen/assert_screen, but it has problem for assert_and_CLICK, that's why this test failed at partition_first_disk.pm with log. Root cause is assert_and_click did not work on click, it only asserted:

21:56:41.9625 8599 MATCH(partitioning_firstdisk-select-hard-disks-one-selected-20170831:0.00)
21:56:42.3036 8593 >>> testapi::_check_backend_response: match=select-hard-disks-one-selected timed out after 90

Reply:
ok, so I still don't think that the "hotkey" tags are a good approach but will merge as is now.